AndroidSDK/src/com/leanplum/internal/Request.java

```diff
@@ -198,7 +199,12 @@ private static void saveRequestForLater(Map<String, Object> args) {
       SharedPreferences preferences = context.getSharedPreferences(
           LEANPLUM, Context.MODE_PRIVATE);
       SharedPreferences.Editor editor = preferences.edit();
-      int count = preferences.getInt(Constants.Defaults.COUNT_KEY, 0);
+      long count;
+      if (LeanplumSQLiteHelper.isDatabaseInitialized()) {
```

Comment:
abstract away access to another class to be easily to change underlaying storage